script/lib/util.py

```diff
@@ -25,12 +25,6 @@ def is_verbose_mode():
 TS_NODE = os.path.join(ELECTRON_DIR, 'node_modules', '.bin', 'ts-node')
 SRC_DIR = os.path.abspath(os.path.join(__file__, '..', '..', '..', '..'))
 
-NPM = 'npm'
-if sys.platform in ['win32', 'cygwin']:
-  NPM += '.cmd'
-  TS_NODE += '.cmd'
```

Comment:
This line isn't dead code, technically. `TS_NODE` is imported and used by `script/release/uploaders/upload.py`, so this would have a functional change on Windows.

Reply:
oops you're right - accidentally deleted it with NPM but this wasn't reported as unused.